eZ/Publish/Core/settings/storage_engines/legacy/search_query_handlers.yml

```diff
@@ -303,6 +304,12 @@ services:
         tags:
             - {name: ezpublish.persistence.legacy.search.gateway.criterion_handler.content}
 
+    ezpublish.persistence.legacy.search.gateway.criterion_handler.content.permission_subtree:
```

Comment:
So this part will most likly need to be reverted to solve https://github.com/ezsystems/ezplatform-solr-search-engine/pull/34